praw/models/reddit/collections.py

```diff
@@ -165,6 +165,27 @@ def update_description(self, description: str):
             data={"collection_id": self.collection_id, "description": description},
         )
 
+    def update_display_layout(self, display_layout: str):
+        """Update the collection's display layout.
+
+        :param display_layout: Either `TIMELINE` or `GALLERY`.
```

Comment:
```suggestion
        :param display_layout: Either `"TIMELINE"` or `"GALLERY"`.
```
Can we also explain what each option does? Also, is it case sensitive?

Reply:
In the Reddit API, they appear in all caps, so I'm assuming it's case-sensitive. I'll be sure to push a description for each.